### PR TITLE
Return value for route-action helper

### LINKED ISSUE #3: Does not return value from route closure action
It would be useful to be able to also have return values when using this helper.

### PR DESCRIPTION
**Pull request by poteto**
This is not an ideal solution, but I wanted to open this PR to get the ball rolling and to add a test.

Closes #3

### CODE REVIEW

**Comment by rwjblue** [comment]
I'll try to review in more detail when I'm back at a computer. 

A few general concerns (that we can add tests for):

* this looks the action up in the current route structure at the time the helper is rendered, and that it may change between then and when it is invoked. 
* _actions is a deprecated property in Ember 2.x (in favor of just using actions directly), so we should use actions and fallback to _actions to avoid the deprecation

**Comment by poteto** [comment]
I addressed your comments, I think

**Comment by rwjblue** [comment]
What about action bubbling by `return true` in an action?  Do we intend to support that also?

File: addon/helpers/route-action.js [hunk 1]
```diff
@@ -18,9 +26,18 @@ export default Helper.extend({
     assert('[ember-route-action-helper] Unable to lookup router', router);
 
     return function(...invocationArgs) {
+      let action;
+      let handler = emberArray(getRoutes(router)).find((route) => {
```
Comment (rwjblue) [inline addon/helpers/route-action.js]: Can you extract this into another shared function (`getRouteWithAction` or something)?

**Comment by poteto** [comment]
Hmm, I'm not sure if we should support that. Wouldn't you just use the ordinary string `action` in that case?

**Comment by rwjblue** [comment]
Ya, I don't actually care if we support it or not, but we need to be deliberate about it.  We should also update the README with a decent explanation (of what we do and do not support).

**Comment by poteto** [comment]
Ya, I'll update and ping you again when it's ready for review!

**Comment by poteto** [comment]
@rwjblue Updated! :eyes: please

**Comment by rwjblue** [comment]
Ok, I'm game to try this. We can always rollback if we hit significant issues...

**Comment by rwjblue** [comment]
@homu r+

**Pull request merged**